server/routes/user.js

```diff
@@ -129,9 +129,14 @@ router.post("/logout", async (req, res) => {
 router.post("/changeprofile", async (req, res) => {
   const info = req.body;
   const id = info.id;
-  const value = info.value;
+  var value = info.value;
```

Comment:
Let's keep it as a const. 

If you're trying to do something like this - look into how to do a ternary in javascript/typescript. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
const value = key === 'phone' ? info.key : Number(info.key)